gannonprudhomme/CSCE-482-KNOW-frontend
# Created frontend skeleton

## Description

Creates the frontend skeleton. Note this intentionally looks bad, as I didn't add any CSS.

## How to test

I don't expect you to be able to genuinely review my code since you haven't done React before. However, I would like you to check it out and confirm that it runs as expected, and maybe mess around with it a bit to understand each part.

## Screenshots

![chrome_6HfIRloQYT](https://user-images.githubusercontent.com/7295783/112183568-9d5a0600-8bbb-11eb-8e37-1f857d925936.png)
